src/desktop/components/record_artwork_view/index.coffee

```diff
@@ -0,0 +1,14 @@
+metaphysics = require '../../../lib/metaphysics.coffee'
+mutation = require './mutation.coffee'
+CurrentUser = require '../../models/current_user.coffee'
+
+module.exports =
+  recordArtworkView: (artworkId) ->
+    return unless CurrentUser.orNull()
+    send = {
+      query: mutation,
+      variables: { artwork_id: artworkId },
+      req: { user: CurrentUser.orNull() } }
+    metaphysics send
+      # We don't need to do anything on success.
+      .catch console.error.bind console
```

Comment:
Why not write in ES6?

Reply:
Oh yea lol good call. I was sleepy and semi on auto-pilot.